README.md

```diff
@@ -1,15 +1,4 @@
-OPX NAS NDI 
-=============
+# opx-nas-ndi
+This repository contains a mediation layer between the NAS software and the SAI API. It isolates NAS software from changes of the SAI API. The NDI functions accept input data structures from NAS modules and converts them to the format accepted by the SAI interfaces. 
```

Comment:
The NDI **functions** accept ... and **convert** (vs. converts) - sorry missed it in my earlier fix.